cpp/modmesh/toggle/toggle.cpp

```diff
@@ -64,13 +64,7 @@ Toggle & Toggle::instance()
 }
 
 SolidToggle::SolidToggle()
-    : m_use_pyside(
-#ifdef MODMESH_USE_PYSIDE
-          true
-#else
-          false
-#endif
-      )
+    : m_use_pyside(true)
```

Comment:
The reason of I leaves `m_use_pyside` here is modmesh may can select different UI backend in the future, we can use toggle system to select different backend.

Reply:
You read my mind.